src/parser/monk/mistweaver/modules/talents/Lifecycles.tsx

```diff
@@ -78,15 +78,15 @@ class Lifecycles extends Analyzer {
     // Checking for chiji stacks and determine mana reduction
     const chijiStacksAtEnvCast = this.selectedCombatant.getBuff(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_BUFF.id, event.timestamp)?.stacks;
     if (!chijiStacksAtEnvCast) {
-      this.calculateEnvManaSaved(SPELLS.ENVELOPING_MIST.manaCost);
+      this.calculateEnvManaSaved(SPELLS.ENVELOPING_MIST.manaCost ? SPELLS.ENVELOPING_MIST.manaCost : 0);
```

Comment:
If these values are ever null and it defaults to 1 it will just provide inaccurate information making the whole statistic pointless and misleading. Since it wouldn't crash people won't come ask for it to be fixed either and just assume damn lifecycles is literally the worst talent in the game and even playing it to the mas it doesn't save me more than 100 mana on top of then maybe going into peak and making me have to deal with them there